connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -3339,6 +3373,9 @@ ble_error_t Gap::prepare_legacy_advertising_set(const AdvertisingParameters& par
 
 void Gap::setEventHandler(Gap::EventHandler *handler)
 {
+    if (handler == nullptr) {
+        tr_warning("setting event handler to a null pointer");
```

Comment:
I know there's a group tag but gap has more than one handler
```suggestion
        tr_warning("setting Gap event handler to a null pointer");
```